app/consolidator/services/FilePartOutputStage.scala

```diff
@@ -0,0 +1,148 @@
+/*
```

Comment:
Custom Sink implementation to write forms data to multiple files, based on maxPerFileBytes and maxTotalBytes configuration.